### PR TITLE
RadioGroup: The radio button's dot should be centered when the browser window is zoomed out and the compact Material theme is used [T1165339]

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

File: scss/widgets/generic/radioButton/_index.scss [hunk 1]
```diff
@@ -11,61 +11,61 @@
...
+    }
   }
 
   &.dx-state-focused {
-    &:not(.dx-state-active) {
-      .dx-radiobutton-icon::before {
-        border: $generic-radiobutton-border-width solid $radiobutton-focused-border-color;
-      }
+    .dx-radiobutton-icon {
+      border: $generic-radiobutton-border-width solid $radiobutton-focused-border-color;
```
Comment (EugeniyKiyashko) [inline scss/widgets/generic/radioButton/_index.scss]: I think it's enough to write:
`border-color: $radiobutton-focused-border-color;`

**Review by marker-dao — DISMISSED** [review]

File: scss/widgets/generic/radioButton/_index.scss [hunk 2]
```diff
@@ -101,11 +101,3 @@ $generic-radiobutton-inner-size: $generic-radiobutton-size - $generic-radiobutto
     }
   }
 }
-
-.dx-rtl .dx-radiobutton,
-.dx-rtl.dx-radiobutton {
-  &.dx-radiobutton-checked .dx-radiobutton-icon-dot {
-    margin-right: math.div($generic-radiobutton-inner-size, 4) + $generic-radiobutton-border-width;
```
Comment (EugeniyKiyashko) [inline scss/widgets/generic/radioButton/_index.scss:108]: `$generic-radiobutton-inner-size` variable is not using anymore

File: scss/widgets/material/radioButton/_index.scss [hunk 3]
```diff
@@ -12,6 +12,7 @@
 $material-radiobutton-border-width: 2px;
 $material-radiobutton-inner-size: $material-radiobutton-size - $material-radiobutton-border-width * 2;
```
Comment (EugeniyKiyashko) [inline scss/widgets/material/radioButton/_index.scss]: This variable is not using anymore

File: testing/testcafe/tests/editors/radioGroup/common.ts [hunk 4]
```diff
@@ -64,10 +67,30 @@ test('Radio buttons placed into the template should not be selected after clicki
...
   await checkGroup(thirdChildGroup, false, false, true);
-}).before(async () => createWidget('dxRadioGroup', {
-  items: [{}, {}, {}],
-  itemTemplate: () => ($('<div>') as any).dxRadioGroup({
-    dataSource: [{}, {}, {}],
-    layout: 'horizontal',
-  }),
-}));
+}).before(async () => {
+  await appendElementTo('#container', 'div', 'template');
```
Comment (EugeniyKiyashko) [inline testing/testcafe/tests/editors/radioGroup/common.ts]: Why we should change this test code and add additional element?

File: scss/widgets/generic/radioButton/_index.scss (hunk 2 above)
Comment (alexanderPolosatov) [reply scss/widgets/generic/radioButton/_index.scss:108]: fixed

File: scss/widgets/generic/radioButton/_index.scss (hunk 1 above)
Comment (alexanderPolosatov) [reply scss/widgets/generic/radioButton/_index.scss]: fixed

File: scss/widgets/material/radioButton/_index.scss (hunk 3 above)
Comment (alexanderPolosatov) [reply scss/widgets/material/radioButton/_index.scss]: fixed

File: testing/testcafe/tests/editors/radioGroup/common.ts (hunk 4 above)
Comment (alexanderPolosatov) [reply testing/testcafe/tests/editors/radioGroup/common.ts]: fixed

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**